test/explorer/environmentExplorer.test.ts

```diff
@@ -736,26 +738,27 @@ describe('environmentExplorer', () => {
                 const contractTreeItems: Array<BlockchainTreeItem> = await blockchainRuntimeExplorerProvider.getChildren(allChildren[1]);
                 const installedContractsTree: Array<BlockchainTreeItem> = await blockchainRuntimeExplorerProvider.getChildren(contractTreeItems[0]);
                 installedContractsTree.length.should.equal(5);
-                const installedContractOne: InstantiateCommandTreeItem = installedContractsTree[0] as InstantiateCommandTreeItem;
+                const installedContractOne: InstantiateCommandTreeItem = installedContractsTree[0] as InstalledChainCodeOpsTreeItem;
```

Comment:
This and other instances of InstantiateCommandTreeItem should be InstalledChainCodeOpsTreeItem instead. Only leaving one comment for all of these.